nasa-veda-platform/scientific-computing/custom-environments.qmd

```diff
@@ -0,0 +1,171 @@
+---
+title: How to use a custom environment in the VEDA JupyterHub
+---
+
+The VEDA JupyterHub lists a few pre-configured environments (collections of Python or R libraries and/or other software) as **Server Options** intended to cover most common geospatial data science needs. These
+are effectively pre-built Docker images ready for users to launch.
+
+If you need a different configuration, there are two options for launching a *custom
+environment*:
+
+- **Use some other pre-built, public Docker image** (see
+  [Environment: Other](#environment-other))
+- **Build your own Docker image** (see
+  [Environment: Build your own image](#environment-build-your-own-image))
+
+Each option is covered in the following sections.
+
+## Environment: Other
+
+If you are aware of an existing pre-built, *publicly accessible Docker image* that
+meets your needs, on the **Server Options** page, open the **Environment**
+drop-down list, choose **Other...**, and specify the public identifier of the
+image in the **Custom image** field.
+
+For example, if a Modified Pangeo Notebook environment were not already one of
+the pre-built environments listed, you could use the standard, publicly
+available Pangeo Notebook image yourself, like so:
+
+![Specifying a publicly available Docker image](images/environment-other-custom-image.png)
+
+At a minimum, the **Custom image** field must include both a name and a tag in
+the form `NAME:TAG`.  There is no assumption that a name without a tag implies
+the `latest` tag, so a tag must be supplied.
+
+When the hostname of the image registry is not specified, it defaults to
+`docker.io`, the canonical hostname of the official Docker registry.  Therefore,
+the following 2 image identifiers are equivalent:
+
+* `pangeo/pangeo-notebook:latest` (as shown in the screenshot above)
+* `docker.io/pangeo/pangeo-notebook:latest` (including a registry hostname prefix)
+
+If you wish to use an image hosted in a regsistry other than `docker.io`, you
```

Comment:
should we show a non-docker example? maybe @maxrjones has something stable on quay? or https://nmfs-opensci.github.io/container-images/

Reply:
Sure, I was thinking about that, so I'll grab one from the link you provided.